wit-manifest.json

```diff
@@ -0,0 +1,7 @@
+[
+    {
+        "commit": "a2e36e2c523c66d499810bd52a2956623e2066f6",
```

Comment:
LGTM, and let me tag this commit of api-languages-sifive.git so that it's an "official" released version.

Reply:
This is now blessed as v0.2.0: https://github.com/sifive/api-languages-sifive/releases/tag/v0.2.0

Reply:
Thanks!